modules/apps/object/object-service/src/main/java/com/liferay/object/internal/related/models/ObjectEntry1to1ObjectRelatedModelsProviderImpl.java

```diff
@@ -166,6 +167,16 @@ public int getRelatedModelsCount(
 		return relatedModels.size();
 	}
 
+	@Override
+	public List<Map<String, Object>> getRelatedSystemModels(
+			long foreignKey, ObjectDefinition objectDefinition,
+			ObjectRelationship objectRelationship)
+		throws PortalException {
+
+		return _objectEntryLocalService.getOneToManySystemRelatedObjectEntries(
```

Comment:
could you explain me latter why are you using a method named 1toMany in class named for 1to1? Thanks :smile: